### ISSUE TITLE
Move Storage

### ISSUE THREAD

**Issue by Anshroid** [Anshroid/Prizmchess#3]
Figure out how to store past/future moves (in memory/storage). Will we just use PGN format? Refer to #6

**Comment by Harik-Sodhi** [comment]
Yes -> we want to create files that people can take onto their computer and download and then upload to chess.com or lichess.org and analyse. (or send on discord to the person you were playing)

**Comment by Anshroid** [comment]
Okay, whoever is doing it can assign this to themself when they're ready to code it.
